config/config-sil.json

```diff
@@ -64,6 +64,29 @@
     "yeti_driver": {
         "module": "JsYetiSimulator"
     },
+    "setup": {
+        "config_module": {
+            "localization": true,
+            "setup_simulation": false,
+            "setup_wifi": true
```

Comment:
I think these all of these options should be set to true.
Or at least simulation and localization, wifi might now be supported on all hosts

Reply:
Thanks!